content/python/concepts/built-in-functions/terms/hex/hex.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'hex()'
+Description: 'Converts an integer number to a lowercase hexadecimal string.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`hex()`** function converts an integer number to a hexadecimal string with the prefix `0x`.
+
+## Syntax
+
+An **integer** is the number zero (0), a postive natural number (1, 2, 3, etc.) or a negative integer that includes a minus sign (-1, -2, -3, etc).
```

Comment:
```suggestion
```
You misunderstand what I meant. I want a sentence explaining what the value `integer` is in the code `hex(integer)`. Something like:
```
The value `integer` is the value `hex()` will turn into a hexadecimal string.
```

It should also be after the ` ```pseudo ` code block.